themes/ddbasic/sass/components/view/view.scss

```diff
@@ -0,0 +1,574 @@
+        }
+        // In Default panel layout (Two sidebars, content in middle)
+        .default-panel-layout .primary-content & {
+          width: getColumn(4 of 8);
+        }
+        .default-panel-layout .attachments-wrapper & {
+          width: getColumn(4);
+        }
+      }
+    }
+    .grid-gutter {
+      // Medium grid
+      .grid-medium & {
+        // In Ding frontpage layout & Ding2 site template
+        .frontpage-layout &,
+        .ding2-site-template .ding2-site-template & {
+          width: getGutter();
+        }
+        // In Default panel layout (Two sidebars, content in middle)
+        .default-panel-layout .primary-content & {
+          width: getGutter(8);
+        }
+        .default-panel-layout .attachments-wrapper & {
+          width: getGutter();
+        }
+      }
+    }
+    // List
+    > ul {
+      padding: 0;
+      list-style: none;
+    }
+  }
+
+  &.max-two-rows,
+  &.slide-on-mobile {
+    .more-link {
+      @include media($mobile) {
+        margin-top: -54px;
+      }
+    }
+  }
+}
+
+// ==========================================================================
+// View ding news
+// ==========================================================================
+
+.view-ding-news {
+  // First child large
+  &.first-child-large {
+    .view-content {
+      .views-row {
+        // First child
+        &:first-child {
+          // Medium grid 
+          .grid-medium & {
+            // In Ding frontpage layout
+            .frontpage-layout & {
+              @include span-columns(8);
+              // Tablet
+              @include media($tablet) {
+                width: 100%;
+                margin-right: 0;
+              }
+            }
+            // In Default panel layout (Two sidebars, content in middle)
+            .default-panel-layout .primary-content & {
+              width: 100%;
+              margin-right: 0;
+            }
+          }
+        }
+        // Medium grid
+        .grid-medium & {
+          // In Ding frontpage layout
+          .frontpage-layout & {
+            @include omega-reset(3n, 12);
+            &:nth-child(3n + 2) {
+              margin-right: 0;
+            }
+            &:nth-child(3n + 3) {
+              clear: left;
+            }
+            // Tablet
+            @include media($tablet) {
+              @include omega-reset(2n, 12);
+              &:nth-child(3n + 2) {
+                margin-right: getGutter();
+              }
+              &:nth-child(3n + 3) {
+                clear: none;
+              }
+              &:nth-child(2n + 1) {
+                margin-right: 0;
+              }
+              &:nth-child(2n + 2) {
+                clear: left;
+              }
+            }
+
+            // Mobile
+            @include media($mobile) {
+              width: 100%;
+              margin: 0 !important;
+              clear: none !important;
+              //display: block !important;
```

Comment:
Describe component contents/purpose, why are these styles grouped? What is the common denominator?